lib/insert/insert_refunds.rb

```diff
@@ -66,8 +66,28 @@ def self.with_stripe(charge, h)
     refund_row = Qx.update(:refunds).set(payment_id: payment_row['id']).ts.where(id: refund_row['id']).returning('*').execute.first
     # Update original payment to increment its refund_total for any future refund attempts
     Qx.update(:payments).set("refund_total=refund_total + #{h['amount'].to_i}").ts.where(id: original_payment['id']).execute
+
+    refund = Refund.find(refund_row['id'])
+    refund_payment = Payment.find(payment_row['id'])
+    refund_payment.update(refund: refund)
```

Comment:
What's this line for?

Reply:
I forgot to remove this one, it's not needed!

Reply:
I forgot to remove it, just did it